app/views/applications/show.html.erb

```diff
@@ -1,31 +1,46 @@
 <h1>Application</h1>
 
-    <h3>Applicants Name: <%= @application.applicant_name %></h3>
+<h3>Applicants Name: <%= @application.applicant_name %></h3>
     
-    <h4>Add a Pet to this Application 
-    <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |f| %>
-    <%= f.label :search %>
-    <%= f.text_field :search %>
-    <%= f.submit "Search" %>
+    
+    <% if @application.status == "In progress"  %>
+        <h4>Add a Pet to this Application 
+        <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |f| %>
```

Comment:
Might need to change 'method: :get' to be 'method: :post' or 'method: :patch'. After this need to update the routes.